lib/gems/pending/metadata/MIQExtract/MIQExtract.rb

```diff
@@ -158,10 +158,10 @@ def extract(category, &blk)
   end
 
   def getVMConfig(_c)
-    # Get VM config in XML format
-    config_xml = @target.vmConfig.toXML(true, @target)
-
     begin
+      # Get VM config in XML format
+      config_xml = @target.vmConfig.toXML(true, @target)
```

Comment:
If you added :vmConfig to the  SUPPORTED_METHODS array and it still didn't work, then something else is wrong, because that SHOULD work. You need to figure out why it's not working.